frontend/public/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
It might be nice to remember that some of these show up in e.g. Slack or the Crisis Mappers SBTF group and when people share particular projects, they might want this to represent the project details in the snippet. I think `React-Helmet` package helps modify this, but I haven't used it yet.

Reply:
yes, you can use react-helmet to modify your head content from within React component...